File: chatbot.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

console.log("Simple Chatbot Started. Type 'exit' to quit.");

function chatbotResponse(input) {
  const text = input.toLowerCase();

  if (text.includes("hello")) return "Hi! How can I help you?";
  if (text.includes("course")) return "This lab teaches chatbot development.";
  if (text.includes("bye")) return "Goodbye!";

  return "Sorry, I didn't understand that.";
}

function ask() {
  rl.question("You: ", (input) => {

    if (input === "exit") {
      rl.close();
      return;
    }

    console.log("Bot:", chatbotResponse(input));
    ask();
  });
}

ask();

File: Chatbot.java
import java.util.Scanner;

public class Chatbot {

    public static String getResponse(String input) {

        input = input.toLowerCase();

        if (input.contains("hello")) {
            return "Hi! How can I help you?";
        }

        if (input.contains("course")) {
            return "This lab teaches chatbot development using Java.";
        }

        if (input.contains("bye")) {
            return "Goodbye!";
        }

        return "Sorry, I didn't understand that.";
    }

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        System.out.println("Simple Chatbot Started. Type 'exit' to quit.");

        while (true) {

            System.out.print("You: ");
            String userInput = scanner.nextLine();

            if (userInput.equalsIgnoreCase("exit")) {
                System.out.println("Chatbot ended.");
                break;
            }

            String response = getResponse(userInput);
            System.out.println("Bot: " + response);
        }

        scanner.close();
    }
}